meta/documents/user_guide/en/initial_setup.md

```diff
@@ -0,0 +1,121 @@
+# Wayfair plugin: Initial Setup
+
+## Prerequisites
+* [Wayfair API credentials](obtaining_credentials.md).
+
+* Administrative rights on the Plentymarkets system

```

Comment:
This makes sense to me but is "administrative rights" the terms used within PlentyMarkets?

Reply:
The `Access` setting for the user needs to be `Admin`. I will add that detail.

Reply:
@mcloughlin77 I've refactored that area. Please take another look.